study.md

```diff
@@ -21,18 +21,27 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
 ```
+When a nameless function is passed in a function does that callback already have
+to be defined or can it be defined in the body of the root/object function?
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
 ```
+I'm a little unclear about how functions are passed in as arguments so the
+whole code is executed, i.e. rockpaper scissors playToFive had functions as
+arguments.
 
```

Comment:
A function passed to another function is....a callback :) again, hopefully today's material helps.